src/main/java/org/elearning/portfolio/MainApp.java

```diff
@@ -0,0 +1,33 @@
+package org.elearning.portfolio;
+
+import java.util.List;
+
+import org.springframework.context.ApplicationContext;
+import org.springframework.context.support.ClassPathXmlApplicationContext;
+import org.elearning.portfolio.StudentJDBCTemplate;
+
+
+/**
+ * Created by rzonsol on 09.10.2016.
+ */
+
+
+
+public class MainApp {
+    public static void main(String[] args) {
+        ApplicationContext context =
+                new ClassPathXmlApplicationContext("Beans.xml");
```

Comment:
Główny plik konfiguracyjny to "spring-module.xml" i ten zawsze powinniśmy ładować